examples/extension/README.md

```diff
@@ -0,0 +1,12 @@
+# Extension example
+
+This example shows how to create an Elastic Cloud extension using Terraform.
+You can create the extension by `files/content.json`
```

Comment:
I'm not sure what you meant in this sentence, it should be changed to something that guides the user how to create an actual extension, possibly to the Elasticsearch docs?

Reply:
fixed by a55feb8. Please check this README?